Automattic/pocket-casts-ios
# i18n: Add Catalan

Add Catalan:

* To the app
* To the release tooling

## To test

1. Go Schemes > pocketcasts > Options > change "App Language" to "Catalan"
2. Run the app
3. ✅ Check that strings are in Catalan

## Checklist
